includes/functions.php

```diff
@@ -2056,9 +2057,16 @@ function lock_and_purge($table, $sql)
 
         $name = str_replace('_', ' ', ucfirst($table));
         if (is_numeric($purge_days)) {
-            if (dbDelete($table, $sql, [$purge_days])) {
-                echo "$name cleared for entries over $purge_days days\n";
+            if (is_numeric($count)) {
```

Comment:
FYI $count is always numeric, you said it must be an int in the parameters.

Reply:
good catch, removed the check